boards/shields/x_nucleo_nfc04a1/Kconfig.defconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 Kickmaker
```

Comment:
The commit message is wrong.

Reply:
Commit message update to 
```boards: shields: add shield x_nucleo_nfc04a1

create new shield x_nucleo_nfc04a1 with eeprom st25dv
and 3 leds

Signed-off-by: Nathan Olff <nathan@kickmaker.net>
```

Is this ok ?